skimage/util/montage.py

```diff
@@ -108,3 +118,84 @@ def montage2d(arr_in, fill='mean', rescale_intensity=False, grid_shape=None):
     arr_out = arr_out.reshape(alpha_y * height, alpha_x * width)
 
     return arr_out
+
+def montage_rgb(arr_in, fill='mean', grid_shape=None, border_padding=0):
```

Comment:
I agree on this point. `montage2d` and `montage_rgb` perform, basically, the same operation. Could be better to integrate the latter into the former, and add simple check on `ndim` of the input.
Although, if we'd are planning to have `montage_channels` function (which will create a matrix of different channels, not images), we shouldn't merge this functions.